.dockerignore

```diff
@@ -11,6 +11,7 @@
 # Allow on-demand "mvn package".
 !armeria/src/main/**
 !dropwizard/src/main/**
+!jersey2-cassandra3/src/main/**
```

Comment:
@jeqo to have docker build, you need a line similar to this for your kafka example